File: tests/Generator/resource/rust/default/location.rs
use serde::{Serialize, Deserialize};

// Location of the person
#[derive(Serialize, Deserialize)]
pub struct Location {
    #[serde(rename = "lat")]
    lat: Option<f64>,

    #[serde(rename = "long")]
    long: Option<f64>,

}

File: tests/Generator/resource/python/default/location.py
from pydantic import BaseModel, Field, GetCoreSchemaHandler, Tag
from pydantic_core import CoreSchema, core_schema
from typing import Any, Dict, Generic, List, Optional, TypeVar, Annotated, Union, Literal


# Location of the person
class Location(BaseModel):
    lat: Optional[float] = Field(default=None, alias="lat")
    long: Optional[float] = Field(default=None, alias="long")
    pass


